apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -320,7 +320,7 @@ function load_wpcom_block_editor_sidebar() {
  * Coming soon
  */
 function load_coming_soon() {
-	if ( defined( 'WPCOM_PUBLIC_COMING_SOON' ) && WPCOM_PUBLIC_COMING_SOON ) {
+	if ( wpcom_is_proxied_request() ) {
```

Comment:
Do you reckon we should follow a similar approach to the sidebar, where we check for a filter on WPCOM (in the mu-plugin file)? 

E.g., 

```
if ( apply_filters( 'a8c_enable_coming_soon', false ) ) {
    require_once __DIR__ . '/coming-soon/coming-soon.php';
}
```

That way we can enable/disable/gate the feature without having to deploy the ETK?

What do you reckon @p-jackson ?

Reply:
Yes I think we probably should use a filter. It's not just about the advantage being able disable the feature without deploying ETK. `wpcom_is_proxied_request` isn't available in all environments, so it's going to crash if ETK is tested in `wp-env` or (more realistically) loaded on atomic.

Reply:
> Do you reckon we should follow a similar approach to the sidebar, where we check for a filter on WPCOM (in the mu-plugin file)?

hmm ok, yea so for the CfT we will have to enable that on staging I think, that actually has the same behavior of being available to proxied users. 👍 

Yea it looks like the `WPCOM_BLOCK_EDITOR_SIDEBAR` is tested in https://github.com/Automattic/wp-calypso/blob/7cef4ce84239630f99cb12d0198468177fca5aea/apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php#L311 which is the same as the current behavior for WPCOM_PUBLIC_COMING_SOON if I just close this PR